README.md

```diff
@@ -17,17 +17,16 @@ data. And I will have my restore, in this runtime or the next._
 - Product-specific code with simple, testable API
 
 - Complexities to deal with e.g. reuse of blobs with same value in the
-shared code
-    - It is needed to accomplish e.g. fast non-incremental M3 backups that are
```

Comment:
This looks like mutilated second level list

Reply:
Thanks, fixed that.